onnxruntime/core/graph/graph.cc

```diff
@@ -2507,6 +2702,72 @@ std::string Graph::GenerateNodeArgName(const std::string& base_name) {
   return new_name;
 }
 
+static flatbuffers::Offset<flatbuffers::Vector<flatbuffers::Offset<flatbuffers::String>>>
+GetInputsOutputsOrtFormat(flatbuffers::FlatBufferBuilder& builder, const std::vector<const NodeArg*>& src) {
+  std::vector<std::string> vec(src.size());
+  std::transform(src.cbegin(), src.cend(), vec.begin(),
+                 [](const NodeArg* entry) { return entry->Name(); });
+  return builder.CreateVectorOfStrings(vec);
+}
+
+common::Status Graph::SaveToOrtFormat(flatbuffers::FlatBufferBuilder& builder,
+                                      flatbuffers::Offset<fbs::Graph>& fbs_graph) const {
+  auto inputs = GetInputsOutputsOrtFormat(builder, graph_inputs_including_initializers_);
+  auto outputs = GetInputsOutputsOrtFormat(builder, graph_outputs_);
+
+  std::vector<std::string> outer_scope_node_args_vec(resolve_context_.outer_scope_node_args.size());
+  std::copy(resolve_context_.outer_scope_node_args.cbegin(),
```

Comment:
Would be good to add a comment as to why we include these (needed to determine values to pass in when executing a subgraph as we don't have Graph::Resolve in a minimal build so cannot be calculated at runtime)